### PR TITLE
added: Cards

### PR DESCRIPTION
**Pull request by harshalini**
Added Card component in component library
https://epic-banach-ca71de.netlify.app/

### CODE REVIEW

**Review by sks-21 — APPROVED** [review]
The code looks nice. Good to merge
Just one suggestion, you could deploy this repo's dev branch on netlify to get the preview easily.

**Review by munjalSaksham71 — APPROVED** [review]
looks Good. You can merge it

File: components/cards/cards.css [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    margin-top: 3rem;
+    margin-bottom: 1rem;
+}
+.ratings {
+    color: black;
+    position: absolute;
+    margin-top: 6.8rem;
+    margin-left: 1rem;
+    background-color: rgba(255,255,255, 0.7);
+    font-size: 13px;
```
Comment (t007rushi) [inline components/cards/cards.css]: Use rems instead of fixed units like px

File: components/cards/cards.css [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+    margin-bottom: 1.5rem;
+}
+.card-image img {
+    height: 14rem;
+    width: 100%;
+    object-fit: cover;
+    margin-top: 0;
+    position: relative;
+}
+h2 {
```
Comment (t007rushi) [inline components/cards/cards.css]: Use className for good practices

File: components/cards/cards.css [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+button.dismiss-btn {
+    position: absolute;
+    margin-top: -16.2rem;
+    margin-left: 12.5rem;
+    height: 1.25rem;
+    width: 1.25rem;
+    padding: 0;
+    font-size: 10px;
+    border-radius: 50%;
+    background-color: #444444;
```
Comment (t007rushi) [inline components/cards/cards.css]: Use variables for Colors & not directly used it

**Review by t007rushi — CHANGES_REQUESTED** [review]
Important changes to do.

File: components/cards/cards.css (hunk 3 above)
Comment (harshalini) [reply components/cards/cards.css]: Did all the changes, thank you for the suggestions.

**Pull request merged**